src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        String[] fileInfo = workWithFile.readFile(fromFileName).split(System.lineSeparator());
+        String report = workWithFile.createReport(fileInfo);
+        workWithFile.writeFile(report, toFileName);
+    }
+
+    private String readFile(String fromFileName) {
+        File fileWithInfo = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try (FileReader fileReader = new FileReader(fileWithInfo);
+                BufferedReader bufferedReader = new BufferedReader(fileReader)) {
+            String info = bufferedReader.readLine();
+            while (info != null) {
+                stringBuilder.append(info).append(System.lineSeparator());
+                info = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
+        }
+        return stringBuilder.toString();
+    }
+
+    private String createReport(String[] fileInfo) {
+        String[] lineInfo;
+        int numberOfSupply = 0;
+        int numberOfBuy = 0;
+        int totalNumber = 0;
+        StringBuilder report = new StringBuilder();
+        for (String info : fileInfo) {
+            lineInfo = info.split(",");
```

Comment:
Please introduce constant instead of magic value
```suggestion
            lineInfo = info.split(CSV_SEPARATOR);
```